CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ v78
 - Add `segmentPassive` argument to `BinaryenSetMemory` for marking segments
   passive.
 - Make `-o -` print to stdout instead of a file named "-".
+- Change default feature set in the absence of a target features section from
```

Comment:
This should go up above in `Current Trunk`

Reply:
Thanks.